service/rpc2/server.go

```diff
@@ -132,11 +135,48 @@ func (s *RPCServer) Command(command api.DebuggerCommand, cb service.RPCCallback)
 		cb.Return(nil, err)
 		return
 	}
+
+	// don't show the error if we're not on accept-multi mode as we can assume that
+	// the user will have their client connected at all times.
+	if s.config.AcceptMulti {
```

Comment:
do we need to assume the user has their client connected, when we now have `activeClientCounter` ?

Reply:
yup, adressed on the new change. Thanks!